jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ConnectionSettingsMenus.kt

```diff
@@ -174,15 +174,17 @@ class SettingsSelectorComboLabel(private val selectorLogic: SettingsSelectorLogi
     private val label = JBLabel()
 
     init {
+        val arrowLabel = JBLabel(AllIcons.General.ArrowDown)
         add(label, BorderLayout.CENTER)
```

Comment:
ah this should have been addToCenter, doesnt change anything but reads nicer IMO